### DISCUSSION TITLE
Override JPG conversion to PNG

### DISCUSSION THREAD

**Discussion by pothitos** [marp-team/marp#395] [Q&A] [answered]
Hello and thank you again very much for Marp and your prompt answers!

I was trying to embed a big JPG image of 2 MB and export the presentation as PDF. The resulting size of the PDF is 30 MB. If I export it as PPTX, the resulting size is 10 MB.

My guess is that this happens as Marp internally converts JPG to PNG.

In any case, can we reduce the resulting size? Can we just add an image with its original size?

**Comment by yhatt** [comment, marked as answer]
It cannot at this moment. In PDF, there is little that Marp can do because Marp is powered by Chromium browser to make PDF. There are a lot of reports about the same trouble into Chromium bug tracker (crbug). Please track following issues:

- [910588 - Excessive PDF size when background-image doesn't fit print layout height](https://bugs.chromium.org/p/chromium/issues/detail?id=910588)
- [1330530 - Chrome generate PDFs with large-size when the page using the 'obejct-fit:cover' or 'background-size'](https://bugs.chromium.org/p/chromium/issues/detail?id=1330530)
- [1396107 - Unexpectedly high PDF produced by images that should not be converted/reencoded/resized](https://bugs.chromium.org/p/chromium/issues/detail?id=1396107)

A workaround is using PDF reducing tool into the output PDF, such as Ghostscript (https://www.ghostscript.com/blog/optimizing-pdfs.html).

In PPTX, Marp will rasterize each slide pages into PNG, to prevent losing quality for presentation in the whole of document. If it was JPEG, PPTX document would make blurry texts.
